server/src/assignment/assignment.service.ts

```diff
@@ -228,22 +215,35 @@ function computeStatusTab(project, lupteam) {
     return 'archive';
   }
 
-  if (participantProjectMilestones.find(milestone => milestone.statuscode === 'Not Started')) {
-    return 'upcoming';
-  }
-
   // TODO: this is sensitive to sort order. we need to revise this!!!
   if (
-    participantProjectMilestones.find(milestone => ['In Progress', 'Completed'].includes(milestone.statuscode))
-      && participantDispositions.find(disposition => ['Inactive'].includes(disposition.statecode) && ['Submitted', 'Not Submitted'].includes(disposition.statuscode))) {
+    participantDispositions.find(disposition =>
+      // ['General Public', 'LUP'].includes(disposition.dcp_visibility)
```

Comment:
We should remove these commented out lines. Or, if the intention was to clarify what 717170003 and 717170004 represent, we should change the comment to something like:
```
// 717170003 - General Public
// 717170004 - LUP
```
So that the comment isn't confused for commented out code